pkg/apis/serving/v1alpha2/kfservice_framework_tensorrt_test.go

```diff
@@ -52,6 +46,11 @@ var config = FrameworksConfig{
 
 func TestCreateModelServingContainer(t *testing.T) {
 
+	var spec = TensorRTSpec{
```

Comment:
while you are on this, can you help move `requestedResource` and `config` inside test function as well? Thanks!

Reply:
c213948 I moved them too